### PR TITLE
[FSSDK-8951]: Updates readme replacing Full Stack with Feature Experimentation.

### PR DESCRIPTION
**Pull request by yasirfolio3**
## Summary
- Updates readme replacing Full Stack with Feature Experimentation

## Test plan
- All tests should pass

## Issues
- [FSSDK-8951](https://jira.sso.episerver.net/browse/FSSDK-8951)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,19 +6,18 @@
...
 
-This repository houses the Swift SDK for use with Optimizely Full Stack and Optimizely Rollouts for Mobile and OTT.
+This repository houses the Swift SDK for use with Optimizely Feature Experimentation, Optimizely Full Stack (legacy), and Optimizely Rollouts for Mobile and OTT.
 
-Optimizely Full Stack is A/B testing and feature flag management for product development teams. Experiment in any application. Make every feature on your roadmap an opportunity to learn. Learn more at https://www.optimizely.com/platform/full-stack/, or see the [documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/welcome).
+Optimizely Feature Experimentation is an A/B testing and feature flag management tool for product development teams, letting you experiment at every step. Using Optimizely Feature Experimentation allows for every feature on your roadmap to be an opportunity to discover hidden insights. Learn more at [Optimizely.com](https://www.optimizely.com/products/experiment/feature-experimentation/), or see the [developer documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/welcome).
```
Comment (griffincox-optimizely) [inline README.md:11]:
```suggestion
Optimizely Feature Experimentation is an A/B testing and feature management tool for product development teams, letting you experiment at every step. Using Optimizely Feature Experimentation allows for every feature on your roadmap to be an opportunity to discover hidden insights. Learn more at [Optimizely.com](https://www.optimizely.com/products/experiment/feature-experimentation/), or see the [developer documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/welcome).
```

File: README.md [hunk 2]
```diff
@@ -65,6 +64,12 @@ Further installation instructions for Cocoapods: https://guides.cocoapods.org/us
 
 Futher installation instructions for Carthage: https://github.com/Carthage/Carthage
 
+### Feature Management Access
+
+To access the Feature Management configuration in the Optimizely dashboard, please contact your Optimizely account executive.
```
Comment (griffincox-optimizely) [inline README.md:69]:
```suggestion
To access the Feature Management configuration in the Optimizely dashboard, please contact your Optimizely customer success manager.
```

File: README.md (hunk 1 above)
Comment (russell-loube-optimizely) [inline README.md:11]:
```suggestion
Optimizely Feature Experimentation is an A/B testing and feature flag management tool for product development teams, enabling you to experiment at every step. Using Optimizely Feature Experimentation allows for every feature on your roadmap to be an opportunity to discover hidden insights. Learn more at [Optimizely.com](https://www.optimizely.com/products/experiment/feature-experimentation/), or see the [developer documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/welcome).
```

File: README.md [hunk 3]
```diff
@@ -6,19 +6,18 @@
...
 
 
-## Getting Started
+## Get started
 
 ### Using the SDK
 
-See the [Swift SDK developer documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/swift-sdk) to learn how to set
-up an Optimizely project and start using the SDK.
+See the [Swift SDK's developer documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/swift-sdk) for detailed instructions on getting started with using the SDK.
```
Comment (russell-loube-optimizely) [inline README.md:20]:
```suggestion
Refer to the [Swift SDK's developer documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/swift-sdk) for detailed instructions on getting started with using the SDK.
```

File: README.md [hunk 4]
```diff
@@ -6,19 +6,18 @@
...
 
-Optimizely Rollouts is free feature flags for development teams. Easily roll out and roll back features in any application without code deploys. Mitigate risk for every feature on your roadmap. Learn more at https://www.optimizely.com/rollouts/, or see the [documentation](https://docs.developers.optimizely.com/experimentation/v3.1.0-full-stack/docs/introduction-to-rollouts).
+Optimizely Rollouts is [free feature flags](https://www.optimizely.com/free-feature-flagging/) for development teams. You can easily roll out and roll back features in any application without code deploys, mitigating risk for every feature on your roadmap.
 
 
-## Getting Started
+## Get started
 
 ### Using the SDK
```
Comment (russell-loube-optimizely) [inline README.md:18]:
```suggestion
### Use the Swift SDK
```

**Comment by yasirfolio3** [comment]
Suggested changes made.

**Review by griffincox-optimizely — COMMENTED** [review]
1. Change very top "SWIFT SDK" to "Optimizely Swift SDK"
2. Change "Installing the SDK" to "Install the SDK"
3. Change "Other Optimzely SDKs" to "Other Optimizely SDKs"

**Review by griffincox-optimizely — COMMENTED** [review]
LGTM

**Review by jaeopt — APPROVED** [review]
LGTM

**Review by russell-loube-optimizely — COMMENTED** [review]
LGTM - please ask fellow Swift dev to approve.

**Pull request merged**